### PR TITLE
feature(monitor): move to newer images

### PR DESCRIPTION
**Pull request by fruch**
move to formal monitoring images, and skip installation of docker
and such if such images are identifed.
(also mean that now by default we will be using an ubuntu image)

Testing:
- [ ] AWS - https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/longevity/job/longevity-10gb-3h/85/
- [ ] GCE - https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/longevity-10gb-3h-gce-test/24/

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
testing in:
https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/longevity/job/longevity-10gb-3h/80

File: defaults/gce_config.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ gce_project: '' # empty mean default one, can be overwritten to use different on
 
 gce_image_db: '' # so we can use `scylla_version` as needed
 gce_image_loader: 'https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/family/centos-7'
-gce_image_monitor: 'https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/family/centos-7'
+gce_image_monitor: 'https://www.googleapis.com/compute/alpha/projects/skilled-adapter-452/global/images/scylla-monitoring-cloud-ubuntu-4-0-2-2022-09-19t14-39-35z'
```
Comment (fruch) [inline defaults/gce_config.yaml:8]: should try it with v1 as well (even that this is the url google is returning for this image)

File: defaults/aws_config.yaml [hunk 2]
```diff
@@ -10,22 +10,22 @@ instance_type_monitor: 't3.large'
 regions_data:
   us-east-1: # US East (N. Virginia)
     ami_id_loader: 'ami-00c1a16b5136fbb7c' # Loader dedicated AMI v19
-    ami_id_monitor: 'ami-02eac2c0129f6376b' # Official CentOS Linux 7 x86_64 HVM EBS ENA 1901_01
+    ami_id_monitor: 'ami-01a7efd0776f9a4fe' # scylla_monitoring_2022-09-19T14-12-11Z
```
Comment (roydahan) [inline defaults/aws_config.yaml:13]: Which version is it of monitor?
I would like it to be clear so every time we have new monitor version (e.g. 4.1).

@amnonh will send a PR that change the branch name and the images IDs.
(We would be nicer if we could read these IDs from a centralized location like we have in scylla-version).

Comment (fruch) [reply defaults/aws_config.yaml:13]: I think it's 4.2,

but we take the image for the cached dependencies, not for the monitoring version inside.

we still need to preserve the option to install the monitoring from clean image, cause we manger tests.

**Comment by fruch** [comment]
@roydahan for taking the image in GCE, we have some permission issue to handle first, since the image in one project and we running in a different project

File: defaults/aws_config.yaml (hunk 2 above)
Comment (roydahan) [reply defaults/aws_config.yaml:13]: Installing monitoring will be based on cleanOS.
Wherever we do that (e.g new atifact test for monitor or Azure or manager test).

The rest should use the image as is without installing anything.

This is not high priority at the moment, and you don't need to work on it right now.
(This entire PR).

**Comment by fruch** [comment]
> @roydahan for taking the image in GCE, we have some permission issue to handle first, since the image in one project and we running in a different project

permission issue with GCP is handled and we can use those images across project

File: defaults/aws_config.yaml (hunk 2 above)
Comment (fruch) [reply defaults/aws_config.yaml]: Start a thread with @amnonh so next release would have better names to indicate their branch name

**Review by vponomaryov — APPROVED** [review]
LGTM assuming it is tested and works.

**Comment by fruch** [comment]
there a bigger task for handling new monitor images with their own user-script

**Pull request closed without merging**